### PR TITLE
add kubectl to image

### PR DESCRIPTION
**Pull request by tobru**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,19 +12,30 @@ ENV HELM_VERSION=v3.8.1 \
 RUN apk add --no-cache git openssh bash curl gnupg make ca-certificates yq jq
 
 RUN set -x \
+ && cd /tmp \
+ # Helm
  && URL="https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz" \
- && wget -q -O /tmp/helm.tgz "${URL}" \
+ && wget -q -O /tmp/helm.tgz $URL \
```
Comment (ccremer) [inline Dockerfile:18]: why this change?

**Review by ccremer — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (tobru) [reply Dockerfile:18]: There is no need for another quote as I understand shell

Comment (ccremer) [reply Dockerfile:18]: well, the quotes aren't part of the string:
```bash
$ URL="https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz"
$ echo $URL
https://get.helm.sh/helm--linux-amd64.tar.gz
```
so wrapping quotes around "$URL" makes it safer in case the URL contains spaces. Granted, it's not really the case here, but generally it's safer to use quotes around variables.
